config/prod/dian-hm-prod-migration-s3.yaml

```diff
@@ -1,8 +1,8 @@
 # Provision a general S3 bucket or a Synapse External Bucket
 # http://docs.synapse.org/articles/custom_storage_location.html
-template_path: "remote/s3-bucket-v2.yaml"
+template_path: "remote/s3-bucket-v2.j2"
 stack_name: "dian-hm-prod-migration-s3"
-#sceptre_user_data:
+sceptre_user_data:
```

Comment:
does `michael.votaw` have a synapse ID?  if not you'll need to comment this entire thing out.

Reply:
He does not, but that IAM user will only writing S3... we have to do the rest

Reply:
i don't understand your state, what does `only write S3` and `we have to do the rest` mean?

Reply:
This means he will not use Synapse. We will be receive the data and will use a Synapse service account ourselves